src/main/java/frc/robot/RobotContainer.java

```diff
@@ -518,6 +534,58 @@ private void configureAutoCommands() {
     Shuffleboard.getTab("MAIN").add(autoChooser.getSendableChooser());
   }
 
+  private void configureIntakeCommands() {
+    // intake.setDefaultCommand(new IntakeControl(intake));
+
+    // oi.getIntakeManualButton().onTrue(
+    //   Commands.either(
+    //     Commands.runOnce(intake::disableEStop, intake),
+    //     new IntakeEmergencyStop(intake),
+    //     intake::getEStopEnabled
+    //   ));
+
+    // should disable the default command and only run the intake control based off of the
+    // controller input
+    oi.getIntakeManualOverrideSwitch().onTrue(Commands.run(intake::enableManualOverride, intake));
+
+    oi.getManualRunIntakeButton()
+        .and(intake::manualOverrideEnabled)
+        .onTrue(
+            Commands.either(
+                Commands.sequence(
+                    Commands.runOnce(
+                        () -> leds.setIntakeLEDState(IntakeLEDState.INTAKE_MANUALLY_TURNED_OFF)),
+                    Commands.runOnce(intake::turnIntakeOff, intake).withName("TurnIntakeOff")),
+                Commands.sequence(
+                    Commands.runOnce(
+                        () -> leds.setIntakeLEDState(IntakeLEDState.WAITING_FOR_GAME_PIECE)),
+                    Commands.run(intake::intakeGamePiece, intake).withName("IntakeGamePiece")),
+                intake::runningManualIntake));
+
+    oi.getManualIntakeRightOffButton()
```

Comment:
I think you can delete the right and left bindings as you handle the singular intake above.